### ISSUE TITLE
robotv with IPTV Plugin ?

### ISSUE THREAD

**Issue by di-le** [pipelka/roboTV#65]
Hi,

I have no DVB card or SAT>IP on my VDR, but I use the IPTV plugin with this Channels.conf
https://www.vdr-portal.de/forum/index.php?thread/130780-iptv-t-home-magentatv-channels-conf-aktuelle-eintr%C3%A4ge/

Does robotv also work with IPTV plugin or would support be possible?

I tried it once on a MagentaTVStick. I can start recordings, but in the menu I keep getting the message:
"Fehler 
Verbindung zum Server fehlgeschlagen."

> Feb 23 12:22:44 europa vdr[138110]: [138128] Client with ID 1 seems to be disconnected, removing from client list
Feb 23 12:22:40 europa vdr[138110]: [138143] Welcome client 'roboTV:recommend' with protocol version '8' and priority 7
Feb 23 12:22:40 europa vdr[138110]: [138128] Client 192.168.1.29:17551 with ID 1 connected.
Feb 23 12:22:40 europa vdr[138110]: [138128] loading /VDR/etc/plugins/robotv/allowed_hosts.conf
Feb 23 12:22:39 europa vdr[138110]: [138142] Welcome client 'roboTV:dataservice' with protocol version '8' and priority 1
Feb 23 12:22:39 europa vdr[138110]: [138128] Client 192.168.1.29:17039 with ID 0 connected.
Feb 23 12:22:39 europa vdr[138110]: [138128] loading /VDR/etc/plugins/robotv/allowed_hosts.conf

Many thanks for your response
greetings

**Comment by pipelka** [comment]
I guess these are 2 different issues:

- IPTV
- connection problem

**IPTV**

To be honest: I don't know. I've never tried this nor have the possibility to try this.
If your channels deliver a DVB transport stream it should work.

**connection problem**

It's impossible for me to guess what's the cause of the error. The log only shows the 'dataservice' being connected.
Nothing else.
When does the error message appear exactly ?
What's the corresponding log on the server ?

**Comment by di-le** [comment]
Basically, the IPTV channels should contain a DVB stream

On the client, the message "Fehler Verbindung zum Server fehlgeschlagen" and "Information Robo TV verbunden" keeps appearing every few seconds.
I always end up on the setup page. With back I get to the overview of the recordings. After a few seconds, the setup page returns with the error "Connection to server failed".


I didn't even notice it before. The VDR server crashes and restarts when the robotv client accesses the server.

Feb 26 11:00:15 europa vdr[10542]: [10574] No conflicts
Feb 26 11:00:15 europa kernel: roboTV VDR Serv[10574]: segfault at 18 ip 000055ad80938c92 sp 00007efce452f330 error 4 in vdr[55ad80923000+e0000]
Feb 26 11:00:15 europa kernel: Code: 0f 1f 84 00 00 00 00 00 41 54 55 53 48 83 ec 20 48 8d 2d c1 72 13 00 8b 85 f8 10 00 00 85 c0 74 09 80 bf d8 05 00 00 00 74 0e <48> 8b 47 18 48 83 c4 20 5b 5d 41 5c c3 90 48 89 fb 48 8b bf e8 05
Feb 26 11:00:15 europa runvdr[718]: Segmentation fault
Feb 26 11:00:25 europa runvdr[718]: Fr 26. Feb 11:00:25 CET 2021 restarting VDR

https://pastebin.com/nr2sPA2A

**Comment by pipelka** [comment]
Are you using the latest roboTV vdr plugin (0.14.6) ?

**Comment by pipelka** [comment]
And the latest client (0.9.6) ?

**Comment by di-le** [comment]
Yes, I have 0.14.6 on the server and a few days ago on the client I updated from 0.9.3 to 0.9.6.
Today I uninstalled 0.9.6 again and reinstalled it via the Playstore.

**Comment by di-le** [comment]
I started the server with as few plugins as possible. Even there the error occurs if the IPTV plug-in is not loaded.

**Comment by di-le** [comment]
Because of the many "No conflicts" and "Checking conflicts for:" messages I renamed the timers.conf and started it without epgsearch so that no new timers come and now the problem no longer occurs. However, I don't have an option for live television in the menu.

Feb 26 11:00:15 europa vdr[10542]: [10574] No conflicts
Feb 26 11:00:15 europa vdr[10542]: [10574] Checking conflicts for: 1:I-1-11160-11160:2021-03-09:1849:1901:50:99:Sandmann~Unser Sandmännchen~Gutenachtgeschichten (D):<epgsearch><channel>22 - KiKA HD</channel><searchtimer>Unser Sandmännchen</searchtimer><start>1615312140</start><stop>1615312860</stop><s-id>12</s-id><eventid>48281</eventid></epgsearch><epgd><timerid>90</timerid></epgd>

**Comment by pipelka** [comment]
You need to run the 'Live Channels' app and do the TV setup there.

**Comment by pipelka** [comment]
Timers are a good hint. There might be some trouble in handling the timer change messages.
Investigating, ...
